app/[locale]/events.tsx

```diff
@@ -0,0 +1,182 @@
+import { desc, eq } from 'drizzle-orm';
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { BouncyArrowButton, Button } from '~/components/buttons';
+import Heading from '~/components/heading';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '~/components/inputs';
+import { Card, CardContent, CardDescription, CardTitle } from '~/components/ui';
+import { getTranslations } from '~/i18n/translations';
+import { cn, getKeys } from '~/lib/utils';
+import { db, type events as eventsSchema } from '~/server/db';
+import { getS3Url } from '~/server/s3';
+
+export default async function Events({
+  category: currentCategory,
+  locale,
+}: {
+  category:
+    | (typeof eventsSchema.category.enumValues)[number]
+    | 'recents'
+    | 'featured';
+  locale: string;
+}) {
+  const text = (await getTranslations(locale)).Events;
+
+  const events = await db.query.events.findMany({
+    where:
+      currentCategory === 'recents'
+        ? undefined // No filter for recents, get all
+        : currentCategory === 'featured'
+          ? (event) => eq(event.isFeatured, true)
+          : (event) => eq(event.category, currentCategory),
```

Comment:
Nested ternaries should be banned from JS
```suggestion
    where: (event) => {
      if (currentCategory === 'recents') return undefined;
      else if (currentCategory === 'featured') return eq(event.isFeatured, true);
      return eq(event.category, currentCategory);
    },
```